search/docker-compose.yml

```diff
@@ -285,8 +284,6 @@ services:
     restart: always
     depends_on:
       - cvat
-    environment:
-      BASENAME: ${CVAT_BASEPATH}
```

Comment:
We're taking these out because they weren't needed, is that right? Because the cvat build is happening via this docker-compose, we don't need to pass the path down to its build environment